frontend/dynamic-demo-plugin/README.md

```diff
@@ -60,3 +60,33 @@ Following commands should be executed in Console repository root.
    ```
 
 To test a locally built demo plugin image, simply update and re-apply `oc-manifest.yaml`.
+
+
+## Enable the plugin
```

Comment:
While you're in the file, can you update the image section above so it isn't specific to Docker Hub? Here is what we have in our main README:

https://github.com/openshift/console#deploying-a-custom-image-to-an-openshift-cluster